<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>1</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="question">
       <property name="text">
        <string>ex) 얼마나 먼 곳으로 떠나고 싶나요?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>강원도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>경기도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>경상남도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>경상북도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>광주광역시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>대구광역시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>대전광역시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>부천광역시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>울산광역시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>인천광역시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>서울특별시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>세종특별자치시</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>전라남도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>전라북도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>제주특별자치도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>충청남도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>충청북도</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>(시)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
